torch/csrc/jit/register_special_ops.cpp

```diff
@@ -150,6 +150,68 @@ void recursiveStore(
   }
 }
 
+template<bool if_set_requires_grad>
+Operation createTensorFromList(const Node* node) {
+  auto input = node->inputs().at(0);
+  auto elem_type = input->type();
+  while (auto list_type = elem_type->cast<ListType>()) {
+    elem_type = list_type->getElementType();
+  }
+  checkListInputType(elem_type, node);
```

Comment:
`checkListInputType` will still output `torch.tensor` even if it's a call from `as_tensor`